contrib/kubernetes/README.md

```diff
@@ -0,0 +1,17 @@
+# Synapse in Kubernetes
```

Comment:
I think there's various 3rd party repos that have templates or other bits for setting up Synapse in kubernetes -- would it be better for the documentation to point there instead of putting something in contrib which is likely to rot a bit?

Reply:
For example https://gitlab.com/ananace/charts/-/tree/master/charts/matrix-synapse

Reply:
This is true, the example you mentioned provides a fairly complete setup, but also very complex setup. The intention here is to provide something simpler to give insight into what bits to into a minimal working configuration with workers. Based on my own experiences and feedback in synapse admin group on matrix.org there is some demand for this.

Preferably something like this would go where people can find it, for which contrib would be a nice place. Agreed that keeping it up to date is a challenges, but that's the case for all things in contrib.

Reply:
There's also https://github.com/vector-im/ess-starter-edition-core which was recently released.

----

We discussed this as a team today and would be more comfortable with documenting links off to external repositories where others can collaborate on a kubernetes setup.